exporter/datadogexporter/example/example_k8s_manifest.yaml

```diff
@@ -0,0 +1,257 @@
+# This manifest file is meant as an example of how to deloy otel-agent's as daemonset and otel-collector as a standalone service
+# In a way the correctly identifies hostnames for individual k8s nodes. This is meant as an example only and may differ depending
```

Comment:
Something is missing in this sentence, right?

Reply:
ah yup, updated.